docs/application-setup.md

```diff
@@ -0,0 +1,107 @@
+## Application setup
+
+Prior to the implementation of Luigi, you need to set up your application. This document shows you how to set up a web application using the Luigi micro front-end framework.
+
+## Intitial steps
+
```

Comment:
`Follow these steps to integrate Luigi into your web applications:`
Or anything like that -> we need an introduction before going on with the steps.

Reply:
done